src/main/java/dbdr/domain/admin/controller/AdminController.java

```diff
@@ -0,0 +1,53 @@
+package dbdr.domain.admin.controller;
+
+import dbdr.domain.admin.AdminCreateRequest;
+import dbdr.domain.admin.service.AdminService;
+import dbdr.security.model.AuthParam;
+import dbdr.security.model.DbdrAuth;
+import dbdr.security.model.Role;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "[관리자] 서버관리자 (Admin)", description = "서버관리자 정보 조회, 수정")
+@RestController
+@RequestMapping("/${spring.app.version}/admin")
+@RequiredArgsConstructor
+@Slf4j
+public class AdminController {
+
+    private final AdminService adminService;
+
+    @Operation(summary = "서버관리자 정보 조회",security = @SecurityRequirement(name = "JWT"))
```

Comment:
description을 추가하면 좋을 것 같네요.

```suggestion
    @Operation(summary = "서버관리자 정보 조회", description = "모든 서버관리자의 이름을 리스트로 반환합니다", security = @SecurityRequirement(name = "JWT"))

```